calitp/storage.py

```diff
@@ -99,17 +99,9 @@ def make_name_bq_safe(name: str):
     return str.lower(re.sub("[^\w]", "_", name))  # noqa: W605
 
 
-def prefix_bucket(bucket):
```

Comment:
This is used a bunch of other places in data-infra, ex. `airtable_loader_v2` and `create_external_tables`... Will you update those too or do we want to leave this to support those use cases?

Reply:
I am changing those over for Airtable (and RT) jobs, but the external tables use a flag that's just named the same thing